NAS-104280: Sync LDAP field descriptions with UI help text (#1665)
freenas/freenas-docs: userguide/directoryservices.rst

   | Kerberos Principal      | drop-down menu | ✓        | The location of the principal in the keytab created as described in :ref:`Kerberos Keytabs`.        |
   |                         |                |          |                                                                                                     |
   +-------------------------+----------------+----------+-----------------------------------------------------------------------------------------------------+
   | Encryption Mode         | drop-down menu | ✓        | Options for encrypting the LDAP connection:                                                         |
   |                         |                |          |                                                                                                     |
   |                         |                |          | * *OFF:* Do not encrypt the LDAP connection.                                                        |

>    |                         |                |          | * *ON:* SSL encrypts the LDAP connection and uses port :literal:`636`.                              |

Reviewer: Match the tense of the previous entry.
```suggestion
   |                         |                |          | * *ON:* encrypt the LDAP connection with SSL on port :literal:`636`.                                |
```

>    |                         |                |          | * *START_TLS:* STARTTLS encrypts the LDAP connection and uses the default LDAP port :literal:`389`. |

Reviewer: ```suggestion
   |                         |                |          | * *START_TLS:* encrypt the LDAP connection with STARTTLS on the default LDAP port :literal:`389`.   |
```
